transform/relations/relations.py

```diff
@@ -0,0 +1,13 @@
+from relations.process_data import (
+    process_combined_directories,
+    process_organization_project_relations,
+    process_result_relations,
+)
+from schemas.properties.relations import *
+
+
+def process_relations():
+    """"""
+    process_combined_directories(DATA_DIRECTORIES, SELECTED_COLUMNS)
```

Comment:
I would reorganize the flow to see the progress. Please use tqdm. It would be nice to see that relations are being sent and which phase is currently going on with its progress. If that would require a significant refactor, dont do that. In that case use at least logging

Reply:
Added tqdm